.github/workflows/build-standalone.yml

```diff
@@ -0,0 +1,74 @@
+on:
+  release:
+    types: [created]
+
+name: Build Standalone CLI
+
+jobs:
+  build_cli:
+    name: Build CLI on MacOS
+    runs-on: macos-latest
+    env:
+      APPLE_APP_STORE_CONNECT_KEY_ID: ${{ secrets.APPLE_APP_STORE_CONNECT_KEY_ID }}
+      APPLE_APP_STORE_CONNECT_ISSUER_ID: ${{ secrets.APPLE_APP_STORE_CONNECT_ISSUER_ID }}
+      APPLE_CODESIGN_CERT_ID: "F9F7EEE981D7A60F83EBEC36AD675448CA0AB41B"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ^22
+          cache: yarn
+          cache-dependency-path: |
+            yarn.lock
+      - name: Setup environment variables
+        run: |
+          KEYCHAIN_PASSWORD=$(openssl rand -hex 32)
+          echo "::add-mask::$KEYCHAIN_PASSWORD"
+          echo "KEYCHAIN_PASSWORD=$KEYCHAIN_PASSWORD" >> "$GITHUB_ENV"
+      - name: Import certificates and setup Keychain
+        run: |
+          echo "${{ secrets.APPLE_DEVELOPER_ID_CERTIFICATE_BASE64 }}" | base64 --decode > codesign_certificate.p12
+          echo "${{ secrets.APPLE_APP_STORE_CONNECT_PRIVATE_KEY_BASE64 }}" | base64 --decode > app_store_connect_private_key.p8
+
+          security create-keychain -p "$KEYCHAIN_PASSWORD" build.keychain
+          security default-keychain -s build.keychain
+          security unlock-keychain -p "$KEYCHAIN_PASSWORD" build.keychain
+
+          security import codesign_certificate.p12 -k build.keychain -P "${{ secrets.APPLE_DEVELOPER_ID_CERTIFICATE_PASSWORD }}" -T /usr/bin/codesign
+          security set-key-partition-list -S apple-tool:,apple: -s -k "$KEYCHAIN_PASSWORD" build.keychain
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: |
+          echo yarn_dir="$(yarn cache dir)" >> $GITHUB_OUTPUT
+      - uses: actions/cache@v3
+        with:
+          path: |
+            **/node_modules
+            ${{ steps.yarn-cache-dir-path.outputs.yarn_dir }}
+          key: ${{ runner.os }}-yarn-frontend-${{ steps.yarn-cache-dir-path.outputs.yarn_dir }}-${{ hashFiles('**/yarn.lock') }}
```

Comment:
The `setup-node@v4` action will already be taking care of yarn's cache, so you can delete the `yarn-cache-dir-path` step and use this for the caching step:

```
      - uses: actions/cache@v4
        with:
          path: |
            ./node_modules
          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
```